source/features/mark-merge-commits-in-list.tsx

```diff
@@ -35,23 +37,18 @@ const filterMergeCommits = async (commits: string[]): Promise<string[]> => {
 	return mergeCommits;
 };
 
-function getCommitLink(commit: HTMLElement): HTMLAnchorElement | undefined {
-	return $([
-		'a.markdown-title', // Old view style (before November 2023)
-		'.markdown-title a',
-	], commit);
-}
```

Comment:
This was no longer safe nor necessary. It was being used for both positioning and data scraping